src/Synth/OscilGen.cpp

```diff
@@ -269,7 +269,8 @@ const rtosc::MergePorts OscilGen::ports{
 //operations on FFTfreqs
 inline void clearAll(fft_t *freqs, int oscilsize)
 {
-    memset(freqs, 0, oscilsize / 2 * sizeof(fft_t));
+    fft_t zero = 0;
+    std::fill_n(freqs, oscilsize / 2, zero);
```

Comment:
I'd hope this turns into a memset under the hood. C++ standard should guarantee that std::complex<float> is the same as "_complex float" in C and that representation should be identical in memory as float[2]. Either way, it's better to fix the warning than fret about a few cycles difference.

Reply:
I just found an interesting answer about exactly that on [StackOverflow](https://stackoverflow.com/a/51658588). It says we can either use `static_assert(std::is_trivially_copyable<std::complex<float>>::value);` before `memset`, or use `std::fill`, which is opitimized on GCC and clang, but not on MSVC. What do you think after reading this? I'm OK with either solution.

Reply:
Well, as long as GCC/Clang are happy. AFAIK MSVC still hates the codebase due to parts of the rtosc port structure, so it's not really a supported dev platform anyhow.